### PR TITLE
conan-adapter v0.3.1 integration

### PR DESCRIPTION
**Pull request by ChGen**
conan-adapter v0.3.1 integration.

### CODE REVIEW

**Comment by ChGen** [comment]
@olenagerasimova , please review.

**Comment by olenagerasimova** [comment]
@ChGen please, add (as we have for each adapter):
- integration tests for upload and install to/from the adapter
- smoke test for upload and install to/from the adapter
- adapter settings description to wiki

**Comment by ChGen** [comment]
@olenagerasimova , thanks, 
I have integration tests in the conan-adapter repo now. Should I move them all to the artipie main repo, or only some part of them?

**Comment by olenagerasimova** [comment]
@ChGen no, you should not move or copy any tests from the adapter. Tests in the adapter and artipie differ: adapter tests run adapter slice only, when test in artipie run whole artipie image with corresponding adapter settings file and etc. Please, check any repository test in artipie, for python repo or debian repo as an example.

**Comment by ChGen** [comment]
@olenagerasimova , please review. I added Conan integration tests.

**Review by olenagerasimova — COMMENTED** [review]
@ChGen thanks, integration test look good to me. What about wiki and smoke test?

**Comment by ChGen** [comment]
@olenagerasimova , please review; I added smoke test & basic wiki for conan. Also for integration test, I switched to custom ubuntu image for uniform testing environment and arm64 support.

File: .wiki/repositories/conan.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  port: 9300
+  storage:
+    type: fs
+    path: /var/artipie/data/
+```
+
+### Client configuration
+
+```bash
+conan remote add conan-test http://artipie.artipie:9300 False
```
Comment (olenagerasimova) [inline .wiki/repositories/conan.md:21]: I'd suggest to add more detailed description: what is `conan-test`? What `False` mean?

**Review by olenagerasimova — COMMENTED** [review]
@ChGen thanks, just one cosmetic suggestion

**Comment by ChGen** [comment]
@olenagerasimova , please review, I extended wiki.

**Review by olenagerasimova — APPROVED** [review]
@ChGen thanks, approving

**Pull request merged**